src/factory/useProfileLoader/index.js

```diff
@@ -99,7 +95,7 @@ export default ({ geoId, comparisonGeoId, visuals, populationTables }) => {
         let comparisonVisualsData;
         if (profiles.comparison) {
           const { data } = await client.query({
-            query: buildVisualsQuery(visuals, parent),
+            query: buildVisualsQuery(visuals, profiles.parent),
```

Comment:
QQ: Do we know if this will always be the common parent to both `profile` and `comparison`?

Reply:
It gets one parent in hopes they are both same parent.